src/Meziantou.Extensions.Logging.Xunit.v3/XUnitLogger.cs

```diff
@@ -32,8 +32,9 @@ public class XUnitLogger : ILogger
 
     /// <summary>Creates a new logger instance with the specified test output helper.</summary>
     /// <param name="testOutputHelper">The xUnit.net test output helper.</param>
+    /// <param name="options">The logger options [optional].</param>
     /// <returns>A new <see cref="ILogger"/> instance.</returns>
-    public static ILogger CreateLogger(ITestOutputHelper? testOutputHelper) => new XUnitLogger(testOutputHelper, new LoggerExternalScopeProvider(), "");
+    public static ILogger CreateLogger(ITestOutputHelper? testOutputHelper, XUnitLoggerOptions? options = null) => new XUnitLogger(testOutputHelper, new LoggerExternalScopeProvider(), "", options);
```

Comment:
You cannot update the signature of an existing method this way because it's a binary breaking change (and a potential source breaking change).

Intead, you must add a new overload. (same for all signature changes in the PR)